docs/structure.rst

```diff
@@ -123,34 +124,36 @@ Overview of directories and files
 
 * ``src/pure/``: pure interface (mainly used by the LSP server)
 
-   * ``pure.ml``: provide utilities to roll back the state
+  * ``pure.ml``: provide utilities to roll back the state
 
-* ``src/tool/``: tools
+* ``src/tools/``: tools
 
-   * ``external.ml``: call of external tools
-   * ``hrs.ml``: export to the .hrs format of the confluence competition
-   * ``sr.ml``: algorithm for checking subject reduction
-   * ``tree_graphviz.ml``: representation of trees as graphviz files
-   * ``xtc.ml``: export to the .xtc format of the termination competition
+  * ``external.ml``: call of external tools
+  * ``hrs.ml``: export to the .hrs format of the confluence competition
+  * ``sr.ml``: algorithm for checking subject reduction
+  * ``tree_graphviz.ml``: representation of trees as graphviz files
+  * ``xtc.ml``: export to the .xtc format of the termination competition
+  * ``lib.ml``: miscellaneous functions that can be used to interact with
+    lambdapi
```

Comment:
I cannot find any file of this name.

Reply:
Ah indeed, it's part of another branch from which I diverged